### ISSUE TITLE
Radar data format

### ISSUE THREAD

**Issue by stanny880913** [naurril/SUSTechPOINTS#156]
Hello,I want to label only image and radar, is this posible？ and I saw the data format in your readme, radar data type is `.pcd`, but my file is `.xyz`, is it possible to label too? 
thank you~ 

and i add my data into /data , web show my scene1, but there are no frame can choose ! how to fix it!
this is my folder strtrure
.
├── calib
├── camera
│   └── front
├── label
└── radar
    └── front_points

inside front are .png file and inside front_points are .pcd file, there aare pair(ex : 1.png pair 1.pcd)
how can i do to fix!!

**Comment by naurril** [comment]
A `lidar` folder is needed for this tool to work, and only .pcd and .bin (float32,xyzi) file formats are supported. 
you can put your radar files in `lidar` folder, but  you need to convert them into .bin or .pcd, or add support for .xyz.

**Comment by stanny880913** [comment]
> A `lidar` folder is needed for this tool to work, and only .pcd and .bin (float32,xyzi) file formats are supported.
> you can put your radar files in `lidar` folder, but you need to convert them into .bin or .pcd, or add support for .xyz.

So I can see the radar point like LiDAR ? And create bbox on the radar point and it can automatically project to my image? Thank you

**Comment by naurril** [comment]
yes,  you need radar-camera extrinsic calibration matrix to project  boxes to image.
radar points are too sparse maybe the operation won't be as smooth as lidar.

**Comment by BachThanh** [comment]
Hello. I have a question. I have lidar folder with .bin files but I don't know how to generate label folder with json files. Can you tell me with python file to do it. Thank you.

**Comment by stanny880913** [comment]
> 

您好，我把雷達檔案丟到lidar的資料夾內了，所以他沒有辦法直接把點頭應再3D主畫面上嗎？因為我讀取到了但我都沒有看到點的位置，畫面長的如下
![Screenshot from 2023-06-26 15-05-53](https://github.com/naurril/SUSTechPOINTS/assets/56072246/09777d68-140d-4474-8986-b6cda3a63d75)

**Comment by novada0** [comment]
> > 
> 
> 您好，我把雷達檔案丟到lidar的資料夾內了，所以他沒有辦法直接把點頭應再3D主畫面上嗎？因為我讀取到了但我都沒有看到點的位置，畫面長的如下 ![Screenshot from 2023-06-26 15-05-53](https://private-user-images.githubusercontent.com/56072246/248711028-09777d68-140d-4474-8986-b6cda3a63d75.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ4Mjg0MTcsIm5iZiI6MTcxNDgyODExNywicGF0aCI6Ii81NjA3MjI0Ni8yNDg3MTEwMjgtMDk3NzdkNjgtMTQwZC00NDc0LTg5ODYtYjZjZGEzYTYzZDc1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTA0VDEzMDgzN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA1NDFlNGMxZDNkYTU1MGIxOTE0MzI4MmRhMTgzZDdmN2ViOWIxM2EzNDg0MjYwMjFkNjgzM2QxZWZmZGYyNDYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.o0q_A7V_TDkt1YXZI50p04-4-Bj8ZsPBryfABhnyrLI)

请问你解决这个问题了吗？
